frontendResourceBundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -205,23 +205,33 @@ string.GitMachete.BranchOrCommitCellRendererComponent.sync-to-remote-status-text
 
 action.GitMachete.DiscoverAction.text=Discover Branch Layout
 action.GitMachete.DiscoverAction.description=Discover Branch Layout Tree
-action.GitMachete.DiscoverAction.cant-discover-layout-error-title=Can't discover repository layout
-action.GitMachete.DiscoverAction.cant-get-current-repository-error-title=Can't get current git repository
+action.GitMachete.DiscoverAction.cant-discover-layout-error-title=Can''t discover repository layout
+action.GitMachete.DiscoverAction.cant-get-current-repository-error-title=Can''t get current git repository
 action.GitMachete.DiscoverAction.discovered-branch-tree-dialog.empty-table-text=No branch has been discovered
 action.GitMachete.DiscoverAction.discovered-branch-tree-dialog.save-button-text=Save Discovered Layout
 action.GitMachete.DiscoverAction.discovered-branch-tree-dialog.title=Discovered branch layout
 action.GitMachete.DiscoverAction.repository-discover-error-title=Repository discover error
 action.GitMachete.DiscoverAction.write-file-task-title=Writing branch layout file
-action.GitMachete.DiscoverAction.write-file-error-title=Can't write discovered branch layout
+action.GitMachete.DiscoverAction.write-file-error-title=Can''t write discovered branch layout
 
 string.GitMachete.EnhancedGraphTable.empty-machete-file.upper-text=Provided machete file ({0}) is empty.
-string.GitMachete.EnhancedGraphTable.empty-machete-file.lower-text=Discover the repository's branch layout
+string.GitMachete.EnhancedGraphTable.empty-machete-file.lower-text=Discover the repository''s branch layout
```

Comment:
Okay... so this renders correctly as `'`, right? If so (and if the answers to questions above are ` 1=>NO && 2=>YES`), then add a CI script against _single single_ quotes in this file

Reply:
No, It's not rendered correctly. My fault. Corrected on [revamp/user-facing-texts](https://github.com/VirtusLab/git-machete-intellij-plugin/pull/504)